src/main/scala/ai/starlake/extract/BigQueryFreshnessInfo.scala

```diff
@@ -30,13 +30,13 @@ object BigQueryFreshnessInfo {
     val schemaHandler = new SchemaHandler(storageHandler)
     val domains = schemaHandler.domains()
     val tablesFreshnessStatuses = tables.flatMap { case (dsInfo, tableInfos) =>
-      val domain = domains.find(_.getFinalName().equalsIgnoreCase(dsInfo.getDatasetId.getDataset))
+      val domain = domains.find(_.finalName.equalsIgnoreCase(dsInfo.getDatasetId.getDataset))
```

Comment:
Without this, I could not get the final name in the jinjava interpreter but I can try again, maybe It was a problem of typo.

Reply:
After a test, it works with the getter method. I remember why I changed this to finalName. It was in order to have the same experience in the template as the name.

Currently we retrieve name by calling `name()` but for final name it would be `getFinalName()`. I think a finalName doesn't change once it is initialized and may be a lazy var.

What do you think ?